### ISSUE TITLE
FMU export is allowed for encrypted librariese

### ISSUE THREAD

**Issue by niklwors** [OpenModelica/OpenModelica#12053]
### Description

In OMEdit and also with a mos- script it is possible to export models from an encrypted library as binary  and also as source code MU.
This should be forbidden. 

 - OpenModelica Version: OpenModelica v1.22.0 (64-bit)
 - OS: Windows 10, 64 bit

**Comment by casella** [comment]
@niklwors I'm not sure I understand what he problem is. As long as the code is obfuscated, why should this be forbidden? Encrypted models cannot should not be inspected, but the should be usable.

**Comment by niklwors** [comment]
yes this is correct. But sometimes you don't want to be able to export code or FMUs. For example, because FMUs can behave differently than the Modelica model and you can't make sure that everything works as it should. I have also generated a ticket in the SEMLA repository for it.  [SEMLA ticket](https://github.com/modelica/Encryption-and-Licensing/issues/15)

**Comment by adeas31** [comment]
@niklwors as I understand, we wait until the [SEMLA ticket](https://github.com/modelica/Encryption-and-Licensing/issues/15) is fixed. Then we will update OM accordingly, right?

**Comment by niklwors** [comment]
@adeas31 yes, this is ok.
